custom/conf/app.example.ini

```diff
@@ -576,6 +576,22 @@ ROUTER = console
 ;; The routing level will default to that of the system but individual router level can be set in
 ;; [log.<mode>.router] LEVEL
 ;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;; Print request id which parsed from request headers in access log, when access log is enabled.
+;; * E.g:
+;; * In reuqest Header:         X-Request-ID: test-id-123
+;; * Configuration in app.ini:  REQUEST_ID_HEADERS = X-Request-ID
+;; * Print in log:              127.0.0.1:58384 - - [14/Feb/2023:16:33:51 &#43;0800] "test-id-123"
```

Comment:
Oh wait, the `&#43;` should be `+`. There is no HTML, do not use HTML entity.

Reply:
emmm... 
It seems that `template.Execute()` escapes the `+` to the HTML entity.
I will try to fix it

Reply:
> Oh wait, the `&#43;` should be `+`. There is no HTML, do not use HTML entity.
> 
> (And below, many `&#43;`)

I solved the problem.
Use `"text/template"` instead of `"html/template"` and no escaping will occur~
But this PR have been passed, do I keep pushing on my own branch？

Reply:
Sorry, I don't understand why you need text/template or html/template. Aren't these just some comments? Feel free to ignore my comment if that's done by purpose.

Reply:
![image](https://user-images.githubusercontent.com/33891828/220146462-74fd0c08-92ce-4812-b88b-15a1da9ea2cd.png)
In the image, upper part is the output of "html/template", `+` is indeed escaped.
The comment in `app.example.ini` was copied from `access.log` directly. But I did not find this problem at the time.

Reply:
I see, surprise, that's (a very old) bug of old AccessLogger code .... `logTemplate` shouldn't escape the HTML entities .... the logs should be outputted as raw text. I doubt few people ever used such logger so no bug report yet.

I believe the `text/template` is the right answer.

@wolfogre @lunny  how do you think?

Reply:
This may be a previous bug? Need to create a new PR?

https://github.com/go-gitea/gitea/blob/36d1d5fb7871058e510e59b027d5ee32bba43f2b/modules/context/access_log.go#L9

https://github.com/go-gitea/gitea/blob/36d1d5fb7871058e510e59b027d5ee32bba43f2b/modules/context/access_log.go#L30

https://github.com/go-gitea/gitea/blob/36d1d5fb7871058e510e59b027d5ee32bba43f2b/modules/context/access_log.go#L41

Reply:
The bug was introduced by #14475 . Before, it used `text/template` as my expectation. But #14475 used the incorrect `html/template` ...... since then, nobody ever found this bug because the logger is rarely used .....

Reply:
> This may be a previous bug? Need to create a new PR?

It depends. If I was the PR author, I would fix the bug together to save maintainer's time.

Reply:
okey, I get it~thx

Reply:
@sillyguodong Could you also merge #23013 into this PR?